src/Softeq.NetKit.Chat.Data.Persistent/Repositories/IClientRepository.cs

```diff
@@ -14,6 +14,8 @@ public interface IClientRepository
         Task AddClientAsync(Client client);
         Task UpdateClientAsync(Client client);
         Task DeleteClientAsync(Guid clientId);
+        Task DeleteMemberClientsWithThresholdAsync(Guid memberId, int inactiveHoursThreshold);
```

Comment:
Unobvious name from my perspective, rename to something like that DeleteOverThresholdMemberClientsAsync

Reply:
It's fine for me

Reply:
Current name don't reflex that will be deleted all clients with exceeded threshold just threshold.

Reply:
renamed